### PR TITLE
Store a transaction on the transaction pool

### PR DESCRIPTION
**Pull request by TrustHenry**
Store information, inputs, and outputs of received transactions on the tables.
Add endpoint /transaction_received

Related to #221

### CODE REVIEW

File: src/Stoa.ts [hunk 1]
```diff
@@ -828,6 +850,23 @@ class Stoa extends WebService
                     reject(err);
                 }
             }
+            else if (stored_data.type === "transaction")
+            {
+                try
+                {
+                    let tx = Transaction.reviver("", stored_data.data);
+                    let changes = await this.ledger_storage.putTransactionPool(tx);
```
Comment (MichaelKim20) [inline src/Stoa.ts:858]: `changes ` is not used.

**Review by MichaelKim20 — COMMENTED** [review]
Very good.
I think one test code should be added.
A test is required to verify that the database is stored normally.
And I have a question.
The same transaction will not be received more than once, right?

File: src/Stoa.ts (hunk 1 above)
Comment (TrustHenry) [reply src/Stoa.ts:858]: Using the changes, I modified the code in many areas.
Validates the transaction inserted and the number of DB reflections of outputs & inputs

**Comment by TrustHenry** [comment]
> Very good.
> I think one test code should be added.
> A test is required to verify that the database is stored normally.
> And I have a question.
> The same transaction will not be received more than once, right?

Yes, save the same transaction only once.
I will add a new test related to storage.

**Review by MichaelKim20 — APPROVED** [review]
LGTM

**Pull request merged**